recipes/conversation_view/feed_item_row/feed_item_row.vue

```diff
@@ -198,6 +209,29 @@ export default {
         focusout: () => this.setFocus(false),
       };
     },
+
+    listItemClasses () {
+      return [
+        'd-w100p',
+        'd-box-border',
+        'd-ps-relative',
+        'd-px8',
+        { 'd-bgc-secondary-opaque': this.isActive && this.state === DEFAULT_FEED_ROW_STATE },
+        FEED_ROW_STATE_BACKGROUND_COLOR[this.state],
+        'feed-item-row',
+        'd-t',
+        'd-tp-bgc',
+      ];
+    },
+  },
+
+  mounted () {
+    if (!this.fade) {
+      return;
+    }
+    setTimeout(() => {
+      this.$refs.FeedItemRef.$el.classList.remove(FEED_ROW_STATE_BACKGROUND_COLOR[this.state]);
+    }, 1500);
```

Comment:
I'd recommend storing the return value in a data property and calling clearTimeout on it when the component unmounts otherwise it may be possible to get an error if the timer hasn't triggered yet when the component unmounts.

Reply:
Hmm
I put a true value on fade prop and then 1500ms later it says:
Failed to execute 'remove' on 'DOMTokenList': The token provided must not be empty.
 
How I can test this? did I miss sth?

Reply:
i didnt handle the case where state is normal, then no background is applied and hence no fade can occur. Will change sorry!